k8s/agent-injector-rbac.yaml

```diff
@@ -0,0 +1,27 @@
+# This should be the minium RBAC that is needed for users (not operators) to
```

Comment:
I think this rbac is actually something that is needed for the operator who will be deploying the traffic-manager and not users (who won't need this role attached to their account and shouldn't need these permissions).

Reply:
Doh, that comment is off. I'll change it.

Reply:
Hmm, it's already changed. You need to refresh your browser :-)